src/components/Plugin/Gnosis/Block.vue

```diff
@@ -1,5 +1,8 @@
 <template>
   <Block title="Gnosis Impact" v-if="choices.length > 1">
+    <div v-if="loading" class="loading">
```

Comment:
It would be better if we show the content only when it's loaded, now i still can see the block content with empty value when it's loading.